src/charm.py

```diff
@@ -22,13 +23,21 @@
 class MagmaAccessGatewayOperatorCharm(CharmBase):
     """Charm the service."""
 
+    HARDWARE_ID = "Hardware ID"
```

Comment:
Those variable names are confusing. They make it look like the hardware id is "Hardware ID".

Reply:
Yeah, makes sense.
Added `_LABEL` suffix to both.